kalite/distributed/static/js/distributed/content/views.js

```diff
@@ -95,7 +108,7 @@ window.ContentBaseView = BaseView.extend({
 
         this.active = false;
 
-        this.possible_points = 500;
+        this.possible_points = ds.distributed.turn_off_points_for_videos ? 0 : ds.distributed.points_per_video;
```

Comment:
Foundation for Learning Points Equality!

Yeah, no real strong feelings either way, but it seems simpler, and more transparent to end users to have the same max points for every content kind.